### ISSUE TITLE
Add as much Linting as possible for new code

### ISSUE THREAD

**Issue by tunetheweb** [HTTPArchive/data-pipeline#4]
I'm a big fan of Linters. They ensure code consistency, more readable and searchable code and can help avoid bugs.

Implementing Linters early on is much easier than trying to back port it later.

We've used the [GitHub Super-Linter](https://github.com/github/super-linter) in most of our repos - though not all the code is linted in some of the older repos as not got round to running and fixing issues (often cause I'm less familiar with the pipeline code for example!).

Will this repo be where we're going to store the code? If so I'm happy to set the Super-Linter up for that now if you agree?

What languages will we use? The following is what we use in the Web Almanac repo so shall we start with them?

```
          VALIDATE_BASH: true
          VALIDATE_CSS: true
          VALIDATE_HTML: true
          VALIDATE_JAVASCRIPT_ES: true
          VALIDATE_JSON: true
          VALIDATE_MARKDOWN: true
          VALIDATE_PYTHON_PYLINT: true
          VALIDATE_PYTHON_FLAKE8: true
          VALIDATE_SQLFLUFF: true
          VALIDATE_YAML: true
```

**Comment by rviscomi** [comment]
Yeah adding linter support SGTM. I expect most of this pipeline will be Python, JS, and SQL.
